### ISSUE TITLE
IXmlSerializable interface: Missing nullable annotation

### ISSUE THREAD

**Issue by jbe2277** [dotnet/runtime#1238]
The [IXmlSerializable interface](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.Xml/src/System/Xml/Serialization/IXmlSerializable.cs) is missing a nullable annotation for

```C#
XmlSchema GetSchema();    // Actual
XmlSchema? GetSchema();   // Expected
```

See doc: [IXmlSerializable.GetSchema Method](https://docs.microsoft.com/en-us/dotnet/api/system.xml.serialization.ixmlserializable.getschema?view=netcore-3.1).

**Comment by vcsjones** [comment]
The entire framework is in the process or being annotated. The progress is being tracked here: https://github.com/dotnet/corefx/issues/40623

It looks like `System.Private.Xml` hasn't been done yet.

**Comment by StephenBonikowsky** [comment]
Closing, tracked by dotnet/corefx#40623

**Comment by danmosemsft** [comment]
@jeffhandley I assume not 5.0

**Comment by StephenMolloy** [comment]
I believe this was addressed in #41476
